src/RulesMD/Helpers.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Axe.Windows.Rules;
+using System;
+
+namespace RulesMD
+{
+    static class Helpers
+    {
+        public static string GetStandardName(A11yCriteriaId criteriaId)
+        {
+            switch (criteriaId)
+            {
+                case A11yCriteriaId.AvailableActions:
+                    return "Section 508 502.3.10";
```

Comment:
>Section [](start = 28, length = 7)

Can we possibly pull these strings from the RuleSelection assembly? That would give us one less thing to maintain. If the string resources aren't available directly, we could consider adding a public static method to the RuleSelection assembly--pass in an A11yCriteriaId, get back the string description.

Reply:
The hard-coded strings were a hold-over from when this project wasn't included in the axe-windows solution. But now the change has been made to use the RuleSelection resources.